modules/virt-supported-features-for-datavolumes.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-supported-features-for-virtual-storage.adoc
+
+[id="virt-supported-features-for-datavolumes_{context}"]
+= Supported features for {VirtProductName} DataVolumes
+
+.{VirtProductName} DataVolume support matrix
```

Comment:
I'd prefer to use "virtual-storage" instead of "datavolumes" (If you did not intend to highlight DataVolume). On the one hand is to keep terms consistent, on the other hand, introducing new terms as little as possible unless everyone in this field knows it. AFAIK people once did live migration or clone without DataVolumes. Besides, from DataVolume definition https://github.com/kubevirt/containerized-data-importer/blob/master/doc/datavolumes.md , DV is not the same as storage. I heard local DV, shared DV for the first time.

Others LGTM. A precise table.

Reply:
Thanks @qwang1 --- this is good feedback. I had a feeling using DVs was limiting the scope but as I'm still learning the terminology this is an important thing to note. Changed as requested.